image_geometry/test/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 find_package(ament_cmake_gtest REQUIRED)
-find_package(ament_cmake_pytest REQUIRED)
+# TODO(clalancette) Reenable this when tests are ported to ROS 2
```

Comment:
Out of curiosity why @clalancette here?

Reply:
Because @clalancette  ported that package originally and has the more knowledge about the work done for porting it. I can put my name if it's preferred

Reply:
I don't mind. I always thought putting the name of the user in which adds the comment isn't particularly useful. I haven't seen the case where other names are being put in though...

Reply:
My understanding of the name in the TODO was to keep track of who had the most knowledge about the issue so that we can consult them if we address them. In that particular instance I don't think there is much knowledge except "we need python3 opencv module" so I added back my name